pypi-gowin/setup.py

```diff
@@ -0,0 +1,77 @@
+import os
+import subprocess
+import sys
+from setuptools import setup, find_packages
+from setuptools_scm.git import parse as parse_git
+
+def apycula_version():
+    reqs = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze'])
```

Comment:
Please just use `importlib.metadata`.

Reply:
I have not tested this on 3.7 but otherwise... it seems to work.